\documentclass[11pt]{article}
\usepackage{amsmath,amssymb,euscript,mathrsfs}
\jot=3mm
\parskip 0.0in
\parindent 0.4cm
\renewcommand{\baselinestretch}{1.0}

\usepackage[top=1in,bottom=1in,left=1in,right=1in]{geometry}
\usepackage{mathtools}
\usepackage{graphicx}% Include figure files
\usepackage{bm}% bold math
\usepackage[export]{adjustbox}% align figures
\usepackage{cite}

\usepackage{hyperref}
\usepackage[colorinlistoftodos]{todonotes}

\newcommand{\dx}{\ \mathrm{d}}
\def\bbP{\mathbb{P}}
\def\bbZ{\mathbb{Z}}
\def\bbN{\mathbb{N}}
\def\expt#1{{\langle #1 \rangle}}

\usepackage{mathtools}
\mathtoolsset{showonlyrefs}
\def\fixme#1#2{\textbf{[FIXME ({#1}): #2]}}
 

%\renewcommand{\figurename}{\textbf{{Fig.}}}    
%\renewcommand{\thefigure}{\textbf{\arabic{figure}}}
%\renewcommand{\tablename}{\textbf{{Table}}}        
%\renewcommand{\thetable}{\textbf{\arabic{table}}}
%\def\fref#1{\textbf{Fig.~\ref{#1}}}
%\def\tref#1{\textbf{Table~\ref{#1}}}
%\newcommand{\sref}[1]{Section~\ref{#1}}

\input biorxiv_macros.tex

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}

\begin{center}
\textbf{\large [www.ribosome.xyz] A data-platform for analyses and classification of ribosome structures}

\vspace{7mm}



All of the people\textsuperscript{1,2,3}
\\

\end{center}
\vspace{5mm}
{\small
$^{1}$ Department of Mathematics, University of British Columbia, Vancouver, BC V6T 1Z4, Canada\\
$^{2}$ Department of Computer Science, University of British Columbia, Vancouver, BC V6T 1Z4, Canada\\
$^{3}$ Department of Zoology, University of British Columbia, Vancouver, BC V6T 1Z4, Canada
}


%\vspace{5mm}
%UC Berkeley

%\vspace{3mm}
%$^*$To whom correspondence should be addressed:


\vspace{5mm}


\begin{center}
\textbf{Abstract}\\ %(to be submitted before October 15, 200 words max):}\\
\end{center}
    \textbf{ \href{www.ribosome.xyz}{ribosome.xyz} database is a resource that centralizes and organizes structural data associated with the ribosomal complex and provides multiple tools for its analysis.
    A programmatic conversion mechanism based on protein families is implemented to enforce standard ribosomal protein nomenclature.  We thus classify proteins, as well as rRNA and ligands that figure in the PDB depositions and make their individual export as a standalone \textit{.cif} file available through a web-interface. A catalogue of ligands, antibiotics, messenger and transfer RNA present across deposited structures is made available as well as their residue-interfaces with the structure. Conformational analysis of whole universal protein classes is provided and the mechanism is made available to the user for more granular comparisons. Some structural-geometric metrics are applied to classes of proteins and the ribosome exit tunnel. The 3-dimensional definition of the tunnel is made available for export. For most model organisms, analyses are augmented with the residue-level conservation and physio-chemical metrics provided by the Ribovision3.0. We aim to provide a convenient framework for further tackling the compositional and conformational heterogeneity of the ribosome across species as well as to enable data-acquisition for the ribosomal community.}

    
    \\ 
 
\textbf{Keywords:} Ribosome, database, conformational heterogeneity, crystallographic structure file, ribosomal protein, antibiotic, classification, conservation\\


\newpage
\section*{Introduction}

The ribosome is a universal RNA-protein complex that carries out mRNA transcription. The compilation of amino-acids into polypetide chains based on the sequence on the information encoded in the mRNA underlies the Central Dogma of biology. Despite its universal role and common origin in all species, the ribosomal complex is overwhelmingly diverse.

%
dynamical aspects of ribosomal motion at the single particle level, specialization at the cellular and sub-cellular scale,
or evolutionary differences across species.
Conformational, compositional, intra-cellular heterogeneity.

The recent advances in imaging and processing technologies offer a growing amount of structural samples of this divrersity at ever-finer resolutions. To anticipate the volume and heterogeneity of these data and make them more amenable to computational analysis, we make an effort to organize and classify the major individual components of the ribosome.






\section{Classification of Ribosomal Proteins}
%Allow the user to get access to orhtolog families, extract a set of single protein structures forcomparison (e.g.  distance maps, alignment in Chimera and other APIâ€™s)
The roles, dynamism of ribosomal proteins. %Khanh?

In order to enable comprehensive comparison of structures deposited by the community in to the RCSB/PDB, a common ontology is required for comparing proteins and the data associated with them across multiple files (Fig. 1). One solution is to refer to Uniprot accession codes and/or InterPro families of the proteins, but the naming of ribosomal proteins presents a specific obstacle for data integration. Due to historical contingency, many ribosomal proteins from different species were originally assigned the same name, despite being often unrelated in structure and function. To eliminate confusion, a nomenclature has been proposed to standardize known ribosomal protein names and provide a framework for novel ones. While this nomenclature has been mostly adopted in recent structural studies, PFAM families and UniProt database as well as PDB still contain numerous references to earlier naming systems.

\subsection{Mitigating the nomenclature issues}
[Figure1. Conversion mechanism]
Introduce the problem of nomenclature. Briefly, conversion mechanism, its pros and cons. Example of problematic families.
Enabled comparison across structures: conformationas, conservation. Aligned export.




\section{Applications of conservation and divergence scores}
Relevance of conserved sites. Physio-chemical data associated with conserved residues.
%
the gap between evolutionary and functional studies, by understanding how the translational machinery displays the
capacity to structurally evolve,

\subsection{ProteoVision}

Divergence scores and frequencies.
The extent of provided data. 
Residue-level granularity. Available species.


\section{Ribosome exit tunnel}

The ribosome exit tunnel, a subcompartment of the ribosome that contains the nascent polypeptide chain [13]. We recently performed a more general comparative analysis of the exit tunnel [14] that indicates important geometric differences between eukaryotes and prokaryotes, especially at the constriction site region, where eukaryotic tunnels are more narrow than their prokaryotic counterparts.

Role of the tunnel. Significance of the PTC. Provided and desirable characteristics.
The process with MOLE. Mapping of conservation/phys-chem data on the surface walls.

Ribosome exit tunnel and the peptidyl-transferase center are of particular interest in the exploration of the translation process and evolutionary modifications in different species. We gather a selection of exit-tunnel models from the available structures in the hopes of further extending this dataset in the future.

The constitution of the exit tunnel is of interest for evolutionary, physio-chemical and pharmacological reasons. We provide a mechanism to export some preliminary data about the walls of the exit tunnel as well cylinder-centerline of the tunnel as is caputured in a given model.

Three main features are provided at the moment that characterize the tunnel walls:

    Residue profile of the RPs that interface with the tunnel. (Each protein is identified by its new nomenclature (ex. uL4) where is possible and can thus be compared against homologous chains in other structures.The in-chain IDs of the tunnel-interfacing residues are provided for each protein.)

    Nucleotides of the RNA that interface with the tunnel.

    Ligands, ions or small molecules if any are found embedded in the walls of the tunnel.

\section{rRNA}


A centralized resource to search, access and compare individual rRNA strands across a variety of structures

\secton{Curent Limitations and Future Directions}


Current Limitations/Work in progress

Due to the additional peculiarities of the mitochonodrial ribosome structrue, the current version of the database makes no distinction between cystosolic and mitochondiral, chloroplast ribosomes. Hence, the corresponding nomenclature classes(i.e. uL4m or uL4c) are also absent from the proteins repository.

The species classes of certain structures are somewhat ambiguous due to the fact that PDB marks individual proteins, not whole structures as belongign to a certain species and hence if some structures contain proteins from multiple species, multiple species figure in this structure 's profile(ex.)

    Classification of structures according to following charateristics is desirable, but is not yet implemneted:
        Stages of translation cycle
        Large/Small subunits presence
    Classification for ribosomal proteins based on:
        Buried/solvent-exposed/PTC-interfacing position
        Neighboring proteins

\section{Binding sites}

Crystallographic models frequently feature ligands and small molecules that are not intrinsic to the ribosome but are still of great interest whether due to their pharmacological, evolutionary or other import. We provide a residue-level catalogu of ligands, small molecules and antibiotics and their physical neighborhood as well as tools to search for similar molecules across other structures in the database.


The root of interest in the mRNA, tRNA, antibiotics and other ligands(ex. GTP)
[Figure2. Volume growth] 
Though not all PDB ribosome depositions are explicitly focused on providing structural insight into the binding mechanism of some particular molecule to the ribosome, the number of pharmacologically-interesting ligands embedded in the deposited structures closely tracks the recent surge of depositions.  

Ligand-navigation on the website. Bindning interface analyses/export.




\section{Application Architecture}
[Fig 3. Architecture Slide]
\subsection{Enabling Technologies}

The graph database to facilitate navigation of the semantic data associated with the structure. RCSB's GraphQL WebService. MOLE. BioPython. Django rest, react.
\subsection{Approach to data-integration}


\section{Conclusion}

------


\section{Tools and Data}

\begin{enumerate}

\item Structures
\item Ribosomal Proteins
\item Ligands, Antibiotics and Small Molecules
\item Ribosome Exit Tunnel
\item Ribosomal RNA, tRNA and mRNA

\end{enumerate}
    


\section{Homepage Info -- just for leads, mostly rubbish}
Listed here are models of the ribosome deposited to the Protein Data Bank (include 2014). These include different stages of the translation cycle, assembly and separate subunits.


Individual rRNA, mRNA and tRNA strands along with some RNA-like strands are listed here and made available for download and search.

Ligands, antibiotics and small molecules that figure in the listed structures are gathered here. You can search acroos structures in which a given ligand is present or, conversely, see which ligands figure in given structure.

Given that the binding interface of a certain ligand/antibiotic with the structures might be of interest, we export those in a separate(for now) tool.

We are also ignoring ions (most have not been parsed and listed) for the most part for now given their ubiquity in the resolved structures. We still acknolwedge their importance in certain cases ( ex. the recurrent magnesium-binding motif near the PTC) and aim to add those with more deliberation in later work.

\secton{Tools and Analyses}

\subsection{Ribosomal Proteins Classification}

The absence of standardized naming scheme for ribosomal proteins has been an obstacle to investigation of compositional heterogeneity. We implement the recently proposed nomenclature in code and with reference to sequence-derived protein families (PFAM, Interpro).This makes possible investigation of protein-classes across structures and species.
I will also include a downloadable table of mappings between PFAM families and Ban Classes here.

\subsection{Binding Interfaces}

Ribosomal models are frequently resolved containing extraneous molecules such as antibiotics, inhibitors, transport- and messenger- RNA strands, and ligands. Most of these are of potential pharmacological interest. We provide "binding interfaces" that detail the nature of the binding site across structures for each particular ligand. For a number of ribosomal structures ligands are available(ions are filtered out): you can inspect and download a "binding interface" for each ligand.

A binding interface consists of the residue-wise profile of the ligand itself and the [non-ligand] neighbor-residues(Neighbors) that are within the radius of 5 Angstrom of the ligand.

For each residue its ID is provided along with its parent strand as per the source crystallographic file, which is enough to identify a residue uniquely. Each strand or subchain can be separately downloaded via the icon. Furthermore, the individual ligand interface can also be downloaded as a .json report.

\subsection{Exit Tunnel}

The constitution of the exit tunnel is of interest for evolutionary, physio-chemical and pharmacological reasons. We provide a mechanism to export some preliminary data about the walls of the exit tunnel as well cylinder-centerline of the tunnel as caputured by the MOLE cavity-detection algorithm.

Three main features are provided at the moment that characterize tunnel walls:

    Residue-profiles of the ribosomal proteins that interface with the tunnel. (Each protein is identified by its new nomenclature (ex. uL4, where is possible) and can thus be compared against homologous chains in other structures. The in-chain IDs of the tunnel-interfacing residues are provided for each protein.
    Nucleotide IDs in the RNA chains that interface with the tunnel.
    Ligands, ions or small molecules ( if any are found ) embedded in the walls of the tunnel.

Tunnel-Extraction Method:

The tunnel shape is extracted via the MOLE software. We search for residues-neighbors inside the structure that are within 10 Angstrom of each coordinate-step of the resulting tunnel-model.
In development:

We plan to augment these data with conservation scores and physio-chemical profiles kindly provided by the ProteoVision database and Loren William's group at Georgia Insititute of Technology in the course of further development of both databases.

Quantitative methods have been developed for the analysis of the ribosome exit tunnel which is a region intimately involved in the construction of the nascent peptide chain. We plan to extend this to the large subset (200+) of structures in the database that have canonical composition, i.e. where the subunits are well-assembled.

\subsection{Subcomponent Alignment}







\end{document}