10.1038/s41586-020-2447-x
1. molecular mechanism of the proof-reading aminoacyl-tRNAs from the elongation factor EF-Tu, 33 ribosomal states involved -- all between around 3-4A resolution. Seems a good conf. het. candidate 



https://www.researchgate.net/publication/265557937_Structural_basis_for_the_inhibition_of_the_eukaryotic_ribosome,10.1038/nature13737,"
2. ribosome as a major target for small-molecule inhibitors, used 16 Xray structs to reveal different inhibtors' associations with the rna binding sites, mechanisms of inhibition and conformations of particular proteins, but not of the whole structures. Seems too specifc to fall into the conf. het. category

https://pure.mpg.de/rest/items/item_1854600_6/component/file_1854604/content
3. Dynamics of motion during translocation, 13 intermediate models,intersubuni trotationsand and stalk motion. Unequivocally conf. het.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2442718/pdf/nihms53073.pdf
4. Anisomycin sensitivity of H. marismortui from 13 Xray models, conformations involved are those of mutation sites which affect affinities, not of the whole molecules. Unlikely our category


https://www.nature.com/articles/ncomms8251.pdf
5. Attempts to explain misnse errors during translation, ribosome structures in rare conformations where incorrect tRNAs are incorporated into peptidyl-trna-binding site.
11 X-ray structures of the 70S rib. used.
On the fence about this

6. https://www.researchgate.net/publication/234070674_Dissecting_the_in_vivo_assembly_of_the_30S_ribosomal_subunit_reveals_the_role_of_RimM_and_general_features_of_the_assembly_process

Quantitative mass spectrometry and cryo-electron microscopy to characterize the in vivo 30S ribosome assembly intermediates involved. Contrasts *in-vitro* and *in-vivo* assembly.
16S rRNA models all at around 13A reso

Also on the fence about this one, seems to involve conf.het but not explicitly focus on it

https://www.cell.com/action/showPdf?pii=S1097-2765%2805%2901606-0
7. Focuses on the peptide bond formation and possible sources of catalysis at the PTC. 
9 Xray structures of H. marismortui lsu bound to antibiotics at 2.40 A 
Doesn't seem like our case

https://rnajournal.cshlp.org/content/19/12/1791.full.pdf+html
8. The affect of conformations and mutations 16s rRNA in decoding and their susceptibility to streptomycin.
9 X ray diffraction structures of 30S subunit at around 3.5A.
Seems, again, to focus on one particular region involved in decoding, not the structure.

https://www.cell.com/cell/pdf/S0092-8674(16)31172-2.pdf
9. 11 conformations involved in translation initiation and the motions of initation factors therein. All of 30S subunit in state of initiation at around 4.1 A with cryoem. 
Seems like our case.

https://www.pnas.org/content/pnas/111/35/12740.full.pd
10. Disruption of U32–A38 pairing by the frameshift suppresor tRNA whil decoding +1 "supressible codons", its effects on the maintenance of the correct reading frame. 
9 Xray structures of synthetic constructs paused on different codons, all around 3.5 A
Seems to deal with the chemical architecture of the translation site, not our case.

https://www.cell.com/action/showPdf?pii=S0092-8674%2816%2931467-2
11. Different translational GTPase complexes exploit the plasticity of the ribosomal decoding center to differentially remodel ribosomal proteins and RNA. Additional structural snapshots of thetranslation termination pathway reveal the conforma-tional changes that choreograph the accommoda-tion of decoding factors into the peptidyl transferasecenter 
8 "electrom microscopy" structures at around 3.5 A with different factors bending them. 
Positively our case. 




12. https://www.pnas.org/content/early/2016/07/01/1605127113.short?rss=1&utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+Pnas-RssFeedOfEarlyEditionArticles+%28Early+Edition%29

Negamycin and tetracycline's interference with tRNA binding to the aminoacyl-tRNA site on the small 30S ribosomal subunit. Mutations in the E.coli 70s that confer susceptibility to negamycin, add binding sites. 

8 *Xray* structures at 3A of ecoli ribosome with the u1052G mutation  
Not our case.
2016



http://europepmc.org/article/MED/29875412
13. Assembly, formations states  of the eukaryotic ribosome ranging from nuclear to cytoplasmic intermediaries.

7 single-particle em strucutres of the late human pre-40s ribosome at around 3-4A 

Plainly our case,
2018



https://aac.asm.org/content/58/9/5269?cited-by=yes&legid=aac%3BAAC.03389-14v1
14. Streptogramin and its interactions with the 70s Ecoli ribosome captured by X-ray clgraphy.

7 x-ray strucutres of ecoli 70s bound to different antibiotics  

Not our case
2014



10.1038/s41467-019-10608-z
15. 

Structural basis of release-factor activation during translation termination

7 cryo-em strucutres at 2.9-4A in different conformations of both the ribosom and the release factor inside it

Our case
2019




https://www.nature.com/articles/ncomms2346
16.
Streptomycin's disruptive effects on the decoding binding site in T.thermophilus and hence, recognition of cognate and near-cognate stem-loop analogues.

7 x-ray t.thermophilus strucutres at around 3.5 A with disordered RNA anticodons involved

Not our case
2012



https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5745149/
17. 
Sequential assembly of 60S eukaryotic ribosomal subunit and its intermediary steps captured with cryoem, 25 assembly factors mapped in 6 different assembly states. Reveal the order of integration and compaction of the six major 60S domains

7 cryo-em structures of the 60S subunit between 3.3 and 4.5 A

Our case(if conformations involved in assembly, biogenesis are)
2017


https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5840002/
18. 2′-O-methylation in mRNA disrupts tRNA decoding during translation elongation. Highlights how chemical modifications of mRNA tune the dynamics of protein synthesis.

7 xray structures of t.thermo's 30s subunit at around 3.5A

Perturbations in translation dynamics, not our case
2018


https://www.nature.com/articles/s41594-019-0236-8
19. 
Drug-like molecule PF-06446846 (PF846) binds the human ribosome and selectively blocks the translation of a small number of proteins.These results illuminate how a small molecule selectively targets translation by the human ribosome..

7 xray structures of the human rbiosome at 3.5-3.9A with a stalled nascent pep. chain inside.

Inhibitors\modulators, not our case
2019




https://www.pnas.org/content/109/16/6094
20. 
Unseen intermediate substates of the bacterial ribosome during the first phase of translocation, characterized by intermediate intersubunit rotations, L1 stalk positions, and tRNA configurations. The domain rearrangements are described in quantitative terms, Frank et al

6 states of the wildtype ecoli mRNA-tRNA translocation  captured with s.p. electron microscopy at around 11A each

Def. our case
2011


https://pubmed.ncbi.nlm.nih.gov/29733411/
21. Used cryo-EM to study the molecular origins of the accuracy of ribosome-aided recognition of a cognate ternary complex and the accuracy-amplifying role of the monitoring bases A1492, A1493 and G530 of the 16S rRNA.

6 structs of 70S ecoli ribosome complex obtained by single particle em, all around 3A 


Deals with accuracy, speed of decoding rather than structural conformations. Not our case
2017




https://pubmed.ncbi.nlm.nih.gov/22437501/
22. Here we propose an integrated mechanism for decoding based on six X-ray structures of the 70S ribosome determined at 3.1-3.4 Å resolution, modelling cognate or near-cognate states of the decoding centre at the proofreading step. 

Structures obsolete, replaced by 3A xray diff. analogs.

Seems to deal with the monitoring bases, not conf. het. Not our category
2012